cmd/argocd/commands/admin/backup.go

```diff
@@ -363,19 +388,22 @@ func NewImportCommand() *cobra.Command {
 					fmt.Printf("%s/%s %s needs pruning\n", key.Group, key.Kind, key.Name)
 				}
 			}
+			duration := time.Since(tt)
+			fmt.Printf("import process completed successfully in namespace %s at %s, duration: %s\n", namespace, time.Now().Format(time.RFC3339), duration)
 		},
 	}
 
 	clientConfig = cli.AddKubectlFlagsToCmd(&command)
 	command.Flags().BoolVar(&dryRun, "dry-run", false, "Print what will be performed")
 	command.Flags().BoolVar(&prune, "prune", false, "Prune secrets, applications and projects which do not appear in the backup")
 	command.Flags().BoolVar(&ignoreTracking, "ignore-tracking", false, "Do not update the tracking annotation if the resource is already tracked")
+	command.Flags().BoolVar(&overrideOnConflict, "override-on-conflict", false, "Override the resource on conflict when updating resources")
 	command.Flags().BoolVar(&verbose, "verbose", false, "Verbose output (versus only changed output)")
 	command.Flags().BoolVar(&stopOperation, "stop-operation", false, "Stop any existing operations")
+	command.Flags().StringSliceVarP(&skipResourcesWithLabels, "skip-resources-with-labels", "", []string{}, "Skip importing resources based on the labels e.g. '--skip-resources-with-labels my-label/example.io, my-other-label/example.io'")
```

Comment:
Is this example for multiple labels correct? I thought it would be `--skip-resources-with-labels my-label/example.io --skip-resources-with-labels my-other-label/example.io`

Reply:
tks, usage is wrong. fixed it
here is the example
```
argocd admin import /var/folders/7g/yf8hhs9j2js98xxzs4qlcq4m0000gr/T/tmpkvu8y8xc/dev-xxx-202501311820.yaml --namespace dev-argocd-x-x --stop-operation --prune --ignore-tracking --override-on-conflict --skip-resources-with-labels app.argoproj-internal.io/skip-resource --skip-resources-with-labels app.argoproj-internal.io/skip-resource2
```